README.md

```diff
@@ -29,4 +29,8 @@ To get started with Leeway, follow these steps:
 - Start the storybook server: `yarn storybook`
 
 ## Contributing
-We welcome contributions from the community to enhance Leeway. If you encounter any issues, have ideas for new features, or want to contribute code improvements, feel free to open an issue or even better raise a PR.
\ No newline at end of file
+We welcome contributions from the community to enhance Leeway. If you encounter any issues, have ideas for new features, or want to contribute code improvements, feel free to open an issue or even better raise a PR.
+
+## Project Links
+-Visit our live website: [Website](https://leeway-staging-f2db3.web.app)
+-Explore our Storybook documentation: [Storybook](https://storybook-2c67d.web.app)
```

Comment:
<img width="666" alt="Screenshot 2023-09-01 at 15 19 09" src="https://github.com/lumos-studio/leeway-web/assets/19279756/4cde6387-058c-4327-a84b-0d86070e33c5">

@Joemonmathew Add a space after `-` to make it a bullet point and it will also bring the second point to the next line.

While editing markdown files in vscode you can use the preview button(<img width="28" alt="Screenshot 2023-09-01 at 15 21 26" src="https://github.com/lumos-studio/leeway-web/assets/19279756/a40bef97-51f8-4b60-bf47-edd3e177bf52">) in the top right corner to view and verify the formatting.